packages/happy-css-modules/package.json

```diff
@@ -52,7 +52,7 @@
   "peerDependencies": {
     "less": "^3.0.0 || ^4.0.0",
     "postcss": "^8.0.0",
-    "sass": "^1.63.3"
+    "sass": "^1.68.0"
```

Comment:
`containingUrl` was added in `sass@1.68.0`.
https://github.com/sass/dart-sass/blob/main/CHANGELOG.md#js-api-9